go.sum

```diff
@@ -601,9 +601,6 @@ honnef.co/go/tools v0.0.0-20180728063816-88497007e858/go.mod h1:rf3lG4BRIbNafJWh
 honnef.co/go/tools v0.0.0-20190102054323-c2f93a96b099/go.mod h1:rf3lG4BRIbNafJWhAfAdb/ePZxsR/4RtNHQocxwk9r4=
 honnef.co/go/tools v0.0.0-20190106161140-3f1c8253044a/go.mod h1:rf3lG4BRIbNafJWhAfAdb/ePZxsR/4RtNHQocxwk9r4=
 istio.io/api v0.0.0-20190515205759-982e5c3888c6/go.mod h1:hhLFQmpHia8zgaM37vb2ml9iS5NfNfqZGRt1pS9aVEo=
-istio.io/api v0.0.0-20190522004026-1dffc8d3d26d/go.mod h1:hhLFQmpHia8zgaM37vb2ml9iS5NfNfqZGRt1pS9aVEo=
```

Comment:
I am not familiar with gomod and the need to remove these lines.  Can someone with gomod experience chime in on this PR?

Reply:
This was removed by 

```
go mod tidy
```

It seems to be superfluous

Reply:
See https://github.com/istio/istio/wiki/Vendor-FAQ for handling dependencies.

You will want to update the vendor directory (which is what is used by the builds) to match the go.mod and go.sum with a `go mod vendor` as doc'd in the wiki.

Reply:
I executed both before committing and pushing
```
go mod tidy
go mod vendor
```

Running these two commands on the current PR branch doesn't modify the istio directory at all.

```
$ git status
On branch isolated-go-binary
Your branch is up to date with 'origin/isolated-go-binary'.

nothing to commit, working tree clean
```

Reply:
@kramerul I think this is how it should be, but can you confirm that `go mod vendor` doesn't change anything? I don't think it should. We should really have a test for this (we used to with dep), I filed https://github.com/istio/istio/issues/14795 to get one added

Reply:
Yes, I can confirm this. See my comment above.